region/norfair/west/Business Center.json

```diff
@@ -332,7 +332,31 @@
     },
     {
```

Comment:
I'll let Maddo review this room, since he is more in-the-loop on what's been going on with it